org.gajim.Gajim.yaml

```diff
@@ -16,11 +16,6 @@ finish-args:
   - --talk-name=org.mpris.MediaPlayer2.*
   # Keyring
   - --talk-name=org.freedesktop.secrets
-  # dconf
```

Comment:
@a-andre while you're around, does Gajim uses gsettings ? in such case, we will have to migrate the user settings after closing the dconf hole. See https://blogs.gnome.org/mclasen/2019/07/12/settings-in-a-sandbox-world/

Reply:
No, Gajim doesn't use gsettings.